src/__experimental__/components/date/date.spec.js

```diff
@@ -367,6 +360,26 @@ describe('Date', () => {
         expect(input.instance().value).toBe(mockDate);
       });
     });
+
+    describe('controlled vs uncontrolled input', () => {
+      it('supports being used as an controlled input via passing of a value prop', () => {
+        wrapper = render({ value: '27rd Feb 01' });
```

Comment:
Just wondering whether this `27rd` is intentional? :slightly_smiling_face: (instead of `27th`)

Reply:
Thanks for fixing this :slightly_smiling_face: